sympy/printing/pretty/tests/test_pretty.py

```diff
@@ -5827,3 +5827,62 @@ def test_issue_10472():
 ⎝⎣0  0⎦  ⎣0⎦⎠\
 """)
     assert upretty(M) == ucode_str
+
+
+def test_MatrixElement_printing():
+    # test cases for issue #11821
+    A = MatrixSymbol("A", 1, 3)
+    B = MatrixSymbol("B", 1, 3)
+    C = MatrixSymbol("C", 1, 3)
+    M = MatrixSymbol("M", 1, 3)
+
+    ascii_str1 = "A_00"
+    ucode_str1 = u("A₀₀")
+    assert pretty(A[0, 0])  == ascii_str1
+    assert upretty(A[0, 0]) == ucode_str1
+
+    ascii_str1 = "3*A_00"
+    ucode_str1 = u("3⋅A₀₀")
+    assert pretty(3*A[0, 0])  == ascii_str1
+    assert upretty(3*A[0, 0]) == ucode_str1
+
+    ascii_str1 = "(-B + A)[0, 0]"
+    ucode_str1 = u("(-B + A)[0, 0]")
+    E = A-B
+    F = C[0, 0]
+    F = F.subs(C, E)
+    assert pretty(F)  == ascii_str1
+    assert upretty(F) == ucode_str1
+
+    ascii_str1 = "(-B + A + M)[0, 0]"
```

Comment:
(And this comment applies to the other tests below, too.) Otherwise things look good to me.